test/unit/core.js

```diff
@@ -197,6 +197,34 @@ QUnit.test( "globalEval execution after script injection (#7862)", function( ass
 	assert.ok( window.strictEvalTest - now < 500, "Code executed synchronously" );
 } );
 
+QUnit.test( "jQuery.append with crossOrigin attribute", function( assert ) {
+	var done = assert.async();
+	assert.expect( 1 );
+
+	var timeout = null;
+
+	Globals.register( "corsCallback" );
+	window.corsCallback = function( response ) {
+		assert.ok( response === true, "Cors" );
+
+		if ( timeout != null ) {
+			window.clearTimeout( timeout );
+		}
+
+		done();
+	};
+
+	var src = baseURL + "mock.php?action=cors";
+	src = src.replace( "localhost", "127.0.0.1" );
```

Comment:
Is this simulating a cross-origin request by fetching a script from `127.0.0.1` at a `localhost` host? If that's the case, please add a comment above this line (with an empty line preceeding the comment).